### PR TITLE
simplified kgraph installation.

### PR DESCRIPTION
**Pull request by aaalgo**
that's the best I can do without creating a .deb package.

### CODE REVIEW

**Comment by erikbern** [comment]
Thanks! Can you rebase on latest master? Sorry it seems like there's some merge conflict

**Comment by aaalgo** [comment]
I'm still learning to use github.  I just pulled your latest master and manually merged it in my fork.  Hope that works.

File: install/kgraph.sh [hunk 1]
```diff
@@ -1,7 +1,11 @@
 git clone https://github.com/aaalgo/kgraph
-sudo apt-get install -y libboost-timer-dev libboost-chrono-dev libboost-program-options-dev libboost-system-dev libboost-python-dev
-cd kgraph
+pushd kgraph
+sudo make deps-ubuntu
 make
+<<<<<<< HEAD
```
Comment (erikbern) [inline install/kgraph.sh]: looks like a merge conflict here – can you resolve manually and resubmit? thanks!

**Comment by erikbern** [comment]
no problem – basically do `git pull --rebase` next time and it will try to put your commit on top of the commits in master.

there's a merge conflict you need to resolve manually, can you please do that and resubmit? thanks

**Comment by erikbern** [comment]
with resubmit i mean just push to your origin

**Comment by aaalgo** [comment]
I've pushed to aaalgo/ann-benchmarks

**Comment by erikbern** [comment]
something is messed up with this one – you probably want to rebase & squash all your ocmmits, then do a force push. Or else just let me know and I can fix it.

I'm struggling to compile kgraph right now without modifying the Makefile though

**Comment by aaalgo** [comment]
Let's give up this pull request for now.  It's not that important.  Please send me the problem you are having with KGraph.

**Pull request closed without merging**